### DISCUSSION TITLE
Bot discernment: Ladder vs Non-Ladder

### DISCUSSION THREAD

**Discussion by jh86** [isid0re/SoloLeveling#385] [Q&A]
Hello again!

I noticed that my single player bots are not attempting to make Insights or Spirits etc as those are Ladder only Runewords, so I see there is definitely some thinking being done. They are however keeping 4os Ebugged armors for Fortitude. Does anyone know where the discernment occurs for a Single Player profile vs a ladder profile to decide what to make? Would prefer that I don't find a gemmed ebug Archon Plate ElSolDolLo ;)

**Comment by isid0re** [comment]
You need to do 3 things.

First go to sololeveling/functions/developer.js then set 
`addLadderRW: true`  

Second follow the directions from this reddit post to enable ladder runewords offline. 

https://www.reddit.com/r/diablo2/comments/4nwwc5/confirmed_working_method_for_ladder_runewords_on/

And finally in the d2bs profile manager under parameters you'll need to add
 `-direct -txt`
